src/AppRoutes.jsx

```diff
@@ -28,6 +29,14 @@ const AppRoutes = () => {
         path="/"
         element={session.info.isLoggedIn ? <Navigate to={path} replace /> : <Home />}
       />
+      <Route
+        path="/signup"
+        element={
+          <SessionProvider>
```

Comment:
As the new user signup can be launched from a case manager's session, we want to set up a separate session for the newly-created pod to act in. This child `SessionProvider` "shadows" the parent `SessionProvider`, hiding the case manager's session from this flow without logging the case manager out.